data/categories.txt

```diff
@@ -14574,7 +14574,7 @@ zh-Hans:香水
 zh-Hant:香水
 
 shop-sewing
-en:Sewing Supplies
+en:Sewing Supplies|Habderdashery
```

Comment:
Spelling mistake: haberdashery :)
would also be good to include 'haberdashers' and just 'sewing':

`en:Sewing Supplies|Sewing|Haberdashery|Haberdashers`

Reply:
might be even better to use e.g.
`en:3Sewing Supplies|4Haberdashery`
usage of those numbers is explained in the beginning of the file